.gitignore

```diff
@@ -9,3 +9,5 @@ composer.lock
 phpunit.xml
 phpcs.xml
 .phpcs.xml
+.phpunit.cache
+.phpunit.result.cache
```

Comment:
```suggestion
*.[Cc]ache
```

Reply:
I personally prefer the more verbose format